### DISCUSSION TITLE
allow dataflow to change operator params

### DISCUSSION THREAD

**Discussion by jhellerstein** [hydro-project/hydro#1384] [Ideas]
Some operators like `source_interval` have a parameter that we might like to change dynamically via dataflow.

**Comment by jhellerstein** [comment]
Very few operators have params other than code blocks or file names. Maybe just implement `source_dynamic_interval` or something?

**Comment by MingweiSamuel** [comment]
(possible to do the specific source_interval example in a roundabout way, in rust, via `Stream`s)
